test/components/granules/granule.js

```diff
@@ -94,18 +188,22 @@ test.serial('CUMULUS-336 Granule file links use the correct URL', function (t) {
 test.serial('Checking granule for size prop', function (t) {
   const { container } = render(
     <Provider store={someStore}>
-    <MemoryRouter>
-    <GranuleOverview
-      dispatch={dispatch}
-      match={match}
-      executions={{}}
-      granules={granules}
-      logs={logs}
-      recoveryStatus={{}}
-      skipReloadOnMount={true}
-      workflowOptions={[]}
-    />
-    </MemoryRouter>
+      <MemoryRouter initialEntries={['/granules/my-granule-id']}>
```

Comment:
This looks like something we could store as a variable and use both here and in the previous assertion?